src/modules/EventsList/EventsList.jsx

```diff
@@ -0,0 +1,55 @@
+import './EventsList.scss';
+
+import React, { PureComponent, PropTypes } from 'react';
+import { StaggeredMotion, spring } from 'react-motion'
+
+import cxHelpers from 'lib/decorators/classNameHelpers';
+import EventTile from 'components/EventTile';
+
+@cxHelpers("EventsList")
+class EventsList extends PureComponent {
+  static propTypes = {
+    isLoading: PropTypes.bool.isRequired,
+    events: PropTypes.array
+  };
+
+  renderEvents(events) {
+    const defaultStyles = events.toJS().map(e => ({s: 0}));
+    const styles = (prevInterpolatedStyles) => prevInterpolatedStyles.map((_, i) => {
```

Comment:
word - I was just testing out the ReactMotion lib, and this is their sample boilerplate. Good call, will extract.